examples/authenticated-api/echo/server/jwt_authenticator.go

```diff
@@ -73,6 +74,11 @@ func Authenticate(v JWSValidator, ctx context.Context, input *openapi3filter.Aut
 	if err != nil {
 		return fmt.Errorf("token claims don't match: %w", err)
 	}
+
+	// Set the user property on the echo context so the handler is able to use the token we generate in here.
+	eCtx := middleware.GetEchoContext(ctx)
+	eCtx.Set("user", token)
```

Comment:
This works, however, let's make it nicer code.

Make a global constant, eg, "JWTClaimsContextKey" and named it "jwt_claims" or sometihng. "user" is awfully generic. We do this kind of stuff all over our produciton code, and this recommendation is based on pain experienced.

Reply:
Done